pkg/drivers/kic/oci/network_create.go

```diff
@@ -145,6 +145,7 @@ func tryCreateDockerNetwork(ociBin string, subnet *network.Parameters, mtu int,
 
 	rr, err := runCmd(exec.Command(ociBin, args...))
 	if err != nil {
+		klog.Errorf("failed to create %s network %s %s with gateway %s and mtu of %d: %v", ociBin, name, subnet.CIDR, subnet.Gateway, mtu, err)
```

Comment:
that's a good point, and on the other hand, i also think that with these changes, it should be very unlikely to happen (especially for the end users that usually do not start a number of instances concurrently - more than ~5), and if that's not the case, perhaps we would want it to surface and be reported so that we can address it?